cmd/store.go

```diff
@@ -109,6 +110,12 @@ func (s *store) DeleteNamespace(namespace string) {
 func (s *store) checkRoleForNamespace(role string, namespace string) bool {
 	ar := s.rolesByNamespace[namespace]
 	for _, r := range ar {
+		re := regexp.MustCompile(r)
```

Comment:
Instead of compiling the regex every time we check for a role here, make it a member of the store struct and compile it on initialization of the store, or when the namespace is registered.